src/shared/components/search.tsx

```diff
@@ -630,6 +637,18 @@ export class Search extends Component<SearchRouteProps, SearchState> {
             )}
           </button>
         </div>
+        <div className="col-auto form-check ms-2 mt-0 h-min d-flex align-items-center">
```

Comment:
> The style looks a bit odd even though I used the same classes as other checkboxes in the code. 

It could be the `d-flex` and `align-items-center` that's doing it. [The Bootstrap docs](https://getbootstrap.com/docs/5.2/forms/checks-radios/#without-labels) don't use any classes to change the `display` property like `d-flex` does, and [other places in this codebase that use `form-check`](https://github.com/search?q=repo%3ALemmyNet%2Flemmy-ui%20form-check&type=code) also don't include `d-flex`.

Reply:
Without those the checkbox is aligned at the top of the search button and not centered. And I also dont see any way in bootstrap docs to use secondary color for the checkbox.